azext_iot/central/params.py

```diff
@@ -64,17 +75,38 @@ def load_central_arguments(self, _):
         context.argument(
             "content",
             options_list=["--content", "-k"],
-            help="Configuration for request. "
-            "Provide path to JSON file or raw stringified JSON. "
-            "[File Path Example: ./path/to/file.json] "
-            "[Stringified JSON Example: {'a': 'b'}] ",
+            help="The device template definition. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example: ./path/to/file.json] [Example of stringified JSON: {<Device Template JSON>}]."
+            " The request body must contain CapabilityModel.",
+        )
+
+    with self.argument_context("iot central device-template create") as context:
+        context.argument(
```

Comment:
device template ID should be at the same root level as device-id since most commands share it, or it should be at the device-template level, with generic help text for them all

Reply:
@digimaun do we want to uplevel this param? Does IoTHub/digital twins/any other Azure IoT need it?

Reply:
It may come into play supporting CLI to [DMR integration](https://github.com/Azure/iot-plugandplay-models) or re-imagining of pnp runtime commands. But in general for platform the terminology is "model" , i.e. device model.

Reply:
Moved device-id to `azure iot central device-template`, leave the created one as description is different.